### ISSUE TITLE
Multiple Sentence Input to KeyBERT

### ISSUE THREAD

**Issue by subhamkhemka** [MaartenGr/KeyBERT#53]
Hi

I would like to provide two sentence input to KeyBERT to extract keywords. 

from keybert import KeyBERT

doc = """
         Supervised learning is the machine learning task of learning a function that
         maps an input to an output based on example input-output pairs. It infers a
         function from labeled training data consisting of a set of training examples.
         In supervised learning, each example is a pair consisting of an input object
         (typically a vector) and a desired output value (also called the supervisory signal). 
         A supervised learning algorithm analyzes the training data and produces an inferred function, 
         which can be used for mapping new examples. An optimal scenario will allow for the 
         algorithm to correctly determine the class labels for unseen instances. This requires 
         the learning algorithm to generalize from the training data to unseen situations in a 
         'reasonable' way (see inductive bias).
      """
kw_model = KeyBERT()
keywords = kw_model.extract_keywords(doc)

In the above example - there is one sentence input let's call it sentence A, I would like to include a Sentence B and generate keywords or phrases after modelling both simultaneously.

Is there a way to do this ?

Thanks,
Subham

**Comment by MaartenGr** [comment]
There is no need to model them both simultaneously. Since we are using pre-trained embeddings, there is no training involved and more data would not improve the results. 

If you want to extract the keywords for 2 sentences, I would advise you to loop over the documents and extract them that way.

**Comment by subhamkhemka** [comment]
okay thanks @MaartenGr
